documentation/conf.py

```diff
@@ -85,15 +78,15 @@
 # directories to ignore when looking for source files.
 # This pattern also affects html_static_path and html_extra_path .
 exclude_patterns = ["_build", "Thumbs.db", ".DS_Store"]
-if flexmeasures_app.config.get("FLEXMEASURES_MODE", "") == "demo":
-    exclude_patterns.append("api/*.rst")
-if flexmeasures_app.config.get("FLEXMEASURES_MODE", "") != "play":
```

Comment:
This reminds me that there is a paragraph about resetting all data on a server, which up until this change was shown only for servers in play mode. There should now be a note there that that feature is only available on servers in play mode.

By the way, that page might be a good basis to a user guide. Also, I see that I need to update that page in my PR on deprecating the rolling parameter (i.e. use of repeated time intervals).

Reply:
Indeed, that whole `api/simulation` tutorial is a good beginning for a general tutorial. I'll make a follow-up ticket ticket for that.